operator/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Main
 
+- [11158](https://github.com/grafana/loki/pull/11158) **btaani**: operator: Add support to add additional storage schema V13
```

Comment:
I think the feature we're adding is the warning nudging the user to upgrade the schema:

```suggestion
- [11158](https://github.com/grafana/loki/pull/11158) **btaani**: operator: Add warning for old schema configuration
```